grakn-test/src/test/java/ai/grakn/test/property/OntologyConceptPropertyTest.java

```diff
@@ -163,6 +162,8 @@ public void whenSettingTheDirectSuper_TheDirectSuperIsSet(
         assumeTrue(sameOntologyConcept(subConcept, superConcept));
         assumeThat((Collection<OntologyConcept>) subConcept.subs(), not(hasItem(superConcept)));
 
+        if(subConcept.isType()) assumeThat(subConcept.asType().sup().instances(), is(empty()));
```

Comment:
I feel we should add the same TODO to do these, since this assumption is temporary.